vmdb/app/models/provider_foreman.rb

```diff
@@ -34,4 +34,35 @@ def build_managers
     build_provisioning_manager unless provisioning_manager
     build_configuration_manager unless configuration_manager
   end
+
+  def self.process_tasks(options)
+    raise "No ids given to process_tasks" if options[:ids].blank?
+    if options[:task] == "refresh_ems"
+      refresh_ems(options[:ids])
+      create_audit_event(options)
+    else
+      options[:userid] ||= "system"
+      raise_exception(options)
+      invoke_tasks_queue(options)
+    end
+  end
```

Comment:
Ugh...do we really want process_tasks?  We don't do that on other EMS things, do we?  If I recall that is very Vm specific and is mostly for cross-region support.  @chessbyte Thoughts here?